src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +789,24 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: Rucio RSE name
```

Comment:
can you mention the data type here as well? E.g.: "string with the Rucio RSE name"